### DISCUSSION TITLE
wie kann ich Weishaupt WTC25A-Nachrichten erfassen?

### DISCUSSION THREAD

**Discussion by zenlight777** [john30/ebusd#1358] [Q&A]
Hallo

Ich habe den ebusd-Adapter für die Verwendung mit meinem Weishaupt WTC-25A-Heizsystem gekauft.
Ich habe J0EK3Rs Repo für Weishaupt-Konfigurationen verwendet.

https://adapter.ebusd.eu/v5-c6/index.en.html
https://github.com/J0EK3R/ebusd-configuration-weishaupt?tab=readme-ov-file

Es liest den Status der Heizung mit Temperatur usw. korrekt.
Ich habe versucht, die Temperatur zu lesen/schreiben, aber es schlägt mit einem Fehler fehl:

ebusctl write -c hc1 Set DHWSetTemp=37.0
ERR: Element nicht gefunden

Also wollte ich die Nachrichten des Heizsystems erfassen.

1. ebusctl grab
2. Zieltemperatur manuell über das Menü der Heizung ändern:
P -> 11 auswählen -> 18 auswählen
-> Temperatur auf 64 Grad ändern
-> Temperatur auf 65 Grad ändern
-> Temperatur auf 66 Grad ändern
3. ebusctl grab result

Aber ich bekomme nichts als Ausgabe.
Meine Frage also: Muss ich irgendetwas Besonderes tun? Zum Beispiel länger warten? Oder soll das so nicht funktionieren?

Danke!

**Comment by Jippy1971** [comment]
Hallo,
ich möchte gerne einige Daten über den eBUS meiner WTC 25-A mit einer "alten" Fernbedienstation WCM-FS auslesen.
Dazu hängt am eBUS zusätzlich ein Adapter Shield C6.
Inzwischen habe ich es geschafft, dass ich zumindest Aussentemperatur, Vorlauftemperatur und Warmwasser auslesen kann.
Ich hätte nun gerne noch (Tages-)Brennerstarts und (Tages-)Brennerbetriebsstunden.
Diese kann ich mir direkt am WTC 25-A in den Infos anzeigen lassen, über eBUS finde ich die aber nicht.
Hat jemand evtl. die gleiche bzw. eine ähnliche Konstellation und kann mir einen Tipp geben?
Vielen Dank vorab.
